common/predictive-text/unit_tests/in_browser/CI.conf.js

```diff
@@ -0,0 +1,188 @@
+module.exports = function(config) {
+  var base = require("./base.conf.js");
+
+  var timeouts = base.client.args[0];
+  var browserStackModifier = 10;
+
+  for(key in timeouts) {
+    if(typeof timeouts[key] == 'number') {
+      timeouts[key] = timeouts[key] * browserStackModifier;
+    }
+  }
+
+  timeouts.mobileFactor = 2; // Extra timeout padding for running on a remote mobile device.
+
+  /*
+   * Definition of utility functions for managing our browser lists.
+   */
+  var mergeLaunchers = function() {
+    var mergedDefs = {};
+    var i;
+    for(i=0; i < arguments.length; i++) {
+      for(var name in arguments[i]) {
+        mergedDefs[name] = arguments[i][name];
+
+        // Necessary for Karma to process it properly.
+        mergedDefs[name].base = 'BrowserStack';
+      }
+    }
+
+    return mergedDefs;
+  }
+
+  var toBrowserList = function(mergedSet) {
+    var list = [];
+    for(var name in mergedSet) {
+      list.push(name);
+    }
+
+    return list;
+  }
+
+  /*
+   * Definition of browser sets possibly relevant for testing.
+   */
+  var CURRENT_MAC_LAUNCHERS = {
+    bs_firefox_mac: {
+      browser: 'firefox',
+      browser_version: '62',
+      os: 'OS X',
+      os_version: 'Mojave'
+    },
+    bs_safari_mac_hs: {
+      browser: 'safari',
+      browser_version: '11.1',
+      os: 'OS X',
+      os_version: 'High Sierra'
+    },
+    bs_safari_mac_m: {
+      browser: 'safari',
+      browser_version: '12',
+      os: 'OS X',
+      os_version: 'Mojave'
+    },
+    bs_chrome_mac: {
+      browser: 'chrome',
+      browser_version: '70.0',
+      os: 'OS X',
+      os_version: 'Mojave'
+    }
+  };
+
+  // BrowserStack does not appear to properly support automated JavaScript testing at this time.
+  // See https://www.browserstack.com/list-of-browsers-and-platforms?product=js_testing for the reference.
+  var CURRENT_IOS_LAUNCHERS = {}; /*{
+    bs_iphoneX: {
+      device: 'iPhone X', // Ideally, we'd use 'iPhone X', but BrowserStack's version is being problematic lately.
+      real_mobile: true,
+      os: 'ios',
+      os_version: '11.0'
+    },
+    bs_ipad6: {
+      device: 'iPad 5th',
+      real_mobile: true,
+      os: 'ios',
+      os_version: '11.0'
+    }
+  };*/
```

Comment:
In general, I tend to delete entirely commented-out code when using a VCS. Leave the comment, however.